awxkit/awxkit/cli/resource.py

```diff
@@ -11,6 +11,7 @@
 CONTROL_RESOURCES = ['ping', 'config', 'me', 'metrics']
 
 DEPRECATED_RESOURCES = {
+    'ad_hoc_commands': 'ad_hoc',
```

Comment:
@wenottingham and I just missed this when adding `tower-cli` alias support cc @squid

Reply:
@squidboylan